modules/ecs/shiva/ecs/components/lua_script.hpp

```diff
@@ -11,7 +11,7 @@ namespace shiva::ecs
     struct lua_script
     {
         lua_script(std::string script_, std::string table_name_) noexcept :
-            script(std::move(script_)), table_name(std::move(table_name_))
+            script(script_.c_str()), table_name(table_name_.c_str())
```

Comment:
Isn't this like, a segfault in the waiting?

Reply:
Was not a sigsegv, was a problem fromm DLL's, finally it's better to use a char* for performance here anyway

Reply:
you're still taking the content of a temporary std::string no?